config/routes.rb

```diff
@@ -3,6 +3,7 @@
     root :to => "posts#index"
     resources :posts, path: '', :only => [:show] do
     	get 'comments', on: :member, to: :show
+      get 'posts/:id(/comments)', on: :collection, to: redirect("#{Refinery::Blog.page_url}/%{id}")
```

Comment:
This way there will be the 301 redirect if someone tries to get the post from old uri